rust/trace/src/lib.rs

```diff
@@ -819,6 +821,23 @@ impl Trace {
         samples.sort_unstable();
         samples
     }
+
+    pub(crate) fn save<P: AsRef<Path>>(
```

Comment:
I think this actually probably _is_ public API now? sorry 😬

Reply:
no problem! is done :)